File: 2656-maximum-sum-with-exactly-k-elements/2656-maximum-sum-with-exactly-k-elements.cpp
class Solution {
public:
    int maximizeSum(vector<int>& nums, int k) {
        sort(nums.begin(),nums.end());
        int sum=0;
        int l=nums.size()-1;
        int tmp=nums[l];
        for(int i=0;i<k;i++){
            sum+=tmp;
            tmp++;
        }
        return sum;
    }
};

File: 2656-maximum-sum-with-exactly-k-elements/2656-maximum-sum-with-exactly-k-elements.py
class Solution:
    def maximizeSum(self, nums: List[int], k: int) -> int:
        summ=0
        nums.sort()
        l=len(nums)
        tmp=nums[l-1]
        for i in range(k):
            summ+=tmp
            tmp+=1
        return summ